e2e/cephfs.go

```diff
@@ -127,13 +137,19 @@ var _ = Describe("cephfs", func() {
 			if cephCSINamespace != defaultNs {
 				err := createNamespace(c, cephCSINamespace)
 				if err != nil {
-					Fail(err.Error())
+					e2elog.Failf("failed to create namespace %s %v", cephCSINamespace, err)
 				}
 			}
 			deployCephfsPlugin()
 		}
-		createConfigMap(cephfsDirPath, f.ClientSet, f)
-		createCephfsSecret(f.ClientSet, f)
+		err := createConfigMap(cephfsDirPath, f.ClientSet, f)
+		if err != nil {
+			e2elog.Failf("failed to create configmap %v", err)
+		}
+		err = createCephfsSecret(f.ClientSet, f)
```

Comment:
s/Cephfs/CephFS

Reply:
am not changing the function name in this PR. this is an existing function name